pycbc/inference/option_utils.py

```diff
@@ -37,27 +38,10 @@
 
 def add_low_frequency_cutoff_opt(parser):
     """Adds the low-frequency-cutoff option to the given parser."""
-    # FIXME: this just uses the same frequency cutoff for every instrument for
-    # now. We should allow for different frequency cutoffs to be used; that
-    # will require (minor) changes to the Likelihood class
-    parser.add_argument("--low-frequency-cutoff", type=float,
-                        help="Low frequency cutoff to use for each IFO's PSD "
-                             "estimate.")
-
-
-def low_frequency_cutoff_from_cli(opts):
-    """Parses the low frequency cutoff from the given options.
-
-    Returns
-    -------
-    dict
-        Dictionary of instruments -> low frequency cutoff.
-    """
-    # FIXME: this just uses the same frequency cutoff for every instrument for
-    # now. We should allow for different frequency cutoffs to be used; that
-    # will require (minor) changes to the Likelihood class
-    instruments = opts.instruments if opts.instruments is not None else []
-    return {ifo: opts.low_frequency_cutoff for ifo in instruments}
+    parser.add_argument("--data-conditioning-low-freq", type=float,
+                        nargs="+", action=MultiDetOptionAction,
+                        metavar='IFO:FLOW', dest="low_frequency_cutoff",
+                        help="Low frequency cutoff for fake strain.")
```

Comment:
This is also needed for PSD estimation (as I've found out the hard way). Change to `Low frequency cutoff of the data. Needed for PSD estimation and when creating fake strain.`)